2019-07-11T20:34:03.929Z - error: uncaughtException: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379 date=Thu Jul 11 2019 15:34:03 GMT-0500 (GMT-05:00), pid=8160, uid=null, gid=null, cwd=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master, execPath=C:\Program Files\nodejs\node.exe, version=v10.16.0, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\server], rss=195817472, heapTotal=104824832, heapUsed=70375736, external=105028045, loadavg=[0, 0, 0], uptime=13549, trace=[column=13, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=RedisClient.flush_and_error, line=152, method=flush_and_error, native=false, column=10, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=RedisClient.on_error, line=194, method=on_error, native=false, column=14, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=null, line=109, method=null, native=false, column=13, file=events.js, function=Socket.emit, line=198, method=emit, native=false, column=20, file=domain.js, function=Socket.EventEmitter.emit, line=448, method=emit, native=false, column=8, file=internal/streams/destroy.js, function=emitErrorNT, line=91, method=null, native=false, column=3, file=internal/streams/destroy.js, function=emitErrorAndCloseNT, line=59, method=null, native=false, column=19, file=internal/process/next_tick.js, function=process._tickCallback, line=63, method=_tickCallback, native=false], stack=[Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379,     at RedisClient.flush_and_error (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:152:13),     at RedisClient.on_error (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:194:10),     at Socket.<anonymous> (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:109:14),     at Socket.emit (events.js:198:13),     at Socket.EventEmitter.emit (domain.js:448:20),     at emitErrorNT (internal/streams/destroy.js:91:8),     at emitErrorAndCloseNT (internal/streams/destroy.js:59:3),     at process._tickCallback (internal/process/next_tick.js:63:19)]
2019-07-11T20:34:03.961Z - error: uncaughtException: Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379 date=Thu Jul 11 2019 15:34:03 GMT-0500 (GMT-05:00), pid=8160, uid=null, gid=null, cwd=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master, execPath=C:\Program Files\nodejs\node.exe, version=v10.16.0, argv=[C:\Program Files\nodejs\node.exe, C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\server], rss=196206592, heapTotal=104824832, heapUsed=70572408, external=105028045, loadavg=[0, 0, 0], uptime=13549, trace=[column=19, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\helpers\redisClient.js, function=null, line=13, method=null, native=false, column=13, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=Command.callback, line=1015, method=callback, native=false, column=29, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=RedisClient.flush_and_error, line=158, method=flush_and_error, native=false, column=10, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=RedisClient.on_error, line=194, method=on_error, native=false, column=14, file=C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js, function=null, line=109, method=null, native=false, column=13, file=events.js, function=Socket.emit, line=198, method=emit, native=false, column=20, file=domain.js, function=Socket.EventEmitter.emit, line=448, method=emit, native=false, column=8, file=internal/streams/destroy.js, function=emitErrorNT, line=91, method=null, native=false, column=3, file=internal/streams/destroy.js, function=emitErrorAndCloseNT, line=59, method=null, native=false, column=19, file=internal/process/next_tick.js, function=process._tickCallback, line=63, method=_tickCallback, native=false], stack=[Error: Error: Redis connection to localhost:6379 failed - connect ECONNREFUSED 127.0.0.1:6379,     at C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\helpers\redisClient.js:13:19,     at Command.callback (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:1015:13),     at RedisClient.flush_and_error (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:158:29),     at RedisClient.on_error (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:194:10),     at Socket.<anonymous> (C:\Users\Alan Meza\Desktop\node\EasyERP_open_source-master\node_modules\redis\index.js:109:14),     at Socket.emit (events.js:198:13),     at Socket.EventEmitter.emit (domain.js:448:20),     at emitErrorNT (internal/streams/destroy.js:91:8),     at emitErrorAndCloseNT (internal/streams/destroy.js:59:3),     at process._tickCallback (internal/process/next_tick.js:63:19)]
